src/main/kotlin/com/kotlin/boardproject/domain/notification/dto/NotificationCreateDto.kt

```diff
@@ -1,8 +1,8 @@
-package com.kotlin.boardproject.dto.notification
+package com.kotlin.boardproject.domain.notification.dto
 
-import com.kotlin.boardproject.common.enums.NotificationType
-import com.kotlin.boardproject.model.Notification
-import com.kotlin.boardproject.model.User
+import com.kotlin.boardproject.global.enums.NotificationType
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end